randovania/games/samus_returns/logic_database/Area 7.txt

```diff
@@ -1406,12 +1479,8 @@ Extra - asset_id: collision_camera_018
   > Door to Spider Boost Tunnel South
       Any of the following:
           # Deal with the Gunzoo by the door
-          Lightning Armor or Defeat Gold Robots
-          All of the following:
-              Combat (Beginner)
-              Any of the following:
-                  Super Missile ≥ 2
-                  Missile ≥ 13 and Phase Drift
+          Lightning Armor or Defeat Gold Autrack
+          Phase Drift and Combat (Beginner)
```

Comment:
I'd revert this since there isn't enough time to kill the Gunzoo with just Missiles, and Supers is tight. Also the simplification is just invalid because it assumes combat with nothing extra.